edalize/templates/libero/libero-project.tcl.j2

```diff
@@ -38,7 +38,7 @@ puts "---------- Executing User TCL script: {{src_file|tcl_file_filter|replace("
 
 # Configure Synthesize tool to use the generated Synplify TCL script
 configure_tool -name {SYNTHESIZE} \
-        -params {SYNPLIFY_TCL_FILE:../../{{name}}-syn-user.tcl}
+        -params {SYNPLIFY_TCL_FILE:../{{name}}-syn-user.tcl}
```

Comment:
Usually this path is used with two ".." because the `tcl` file is in the base and Libero reads from `base/prj/synthesis` so the tcl file is two levels below.

Reply:
I really don't know why it read from one folder below only... maybe a version difference but I doubt they changed this from version to version.

Reply:
I can create small simple test core and compare between libero v2021.1 and v12.5.

Reply:
I have tested it against both v2021.1 and v12.5 now, and there is a difference between them. v12.5 references relative paths to the SYNPLIFY_TCL_FILE against {project root}/synthesis, whereas v2021.1 references it against the {project root}.

Should I add a version setting to the libero target? Which version should that then default to? v12.5 to keep current behavior, or the latest version?

Reply:
Damn, why they change these things. Is it possible to identify which version is running or any internal variable holding the root project path. Maybe instead of referencing thru relative paths (../../blabla), use a ({project_root_or_something}). Maybe both versions have the same variable name.

If not possible, I think we could keep the previous version as default since they might be the most used one.

Reply:
Welcome to the EDA world. These people are insanely inconsistent so it's best to rely as little as possible on their conventions unfortunately :/ As I'm not a Libero user myself I can't offer much help here unfortunately but I'm happy to support whatever you come up with.